### ISSUE TITLE
About loss can not decrease

### ISSUE THREAD

**Issue by Rogerlv51** [AIR-DISCOVER/LODE#3]
I use just 1 GPU and train 1 epoch, num_worker = 8, but the loss will stop decreasing when training after 10%, so what should I change the lr or loss_weights? @Philipflyg thank u.

**Comment by Philipflyg** [comment]
Thank you for your interest in our work!

When use 1 GPU and train 1 epoch, the loss will not converge due to the small batch size and short training time. But the loss will still decrease slowly (as shown in the attached figure, where i use the default configuration and 1 gpu, 1 epoch, 8 num_worker).

You can try to train the model with more GPU and/or more epochs. Or tune the hyper-parameters like increasing the lr (but i cannot gaurantee the performance).

![OZJYNM$889$D)QXO7 SCZMT](https://github.com/AIR-DISCOVER/LODE/assets/59141303/91f7368e-8e67-401e-857f-b64501048b8a)
